package.nls.json

```diff
@@ -1,4 +1,5 @@
 {
+  "command.activateIcons.title": "Activate VSCode Icon Theme",
```

Comment:
And in our case the command will show `Icons: Activate VSCode Icon Theme`. `Icons` is already mentioned implicitly.